test/scenarios/annotations/README.md

```diff
@@ -0,0 +1,77 @@
+# Annotated Requests
```

Comment:
Could we link this from the top level README or somewhere else that is typically seen by a developer? The content living here is fine since its relative but I want to ensure it's easy to find for developers.